templates/field/field--node--oe-author--oe-news.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * News author field template.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}{{ item.content }}{{ not loop.last ? ' | ' }}{% endfor %}
```

Comment:
I decided to write it on one line to get rid of new line characters. It is helpful to write test cases for FieldListAssert.